### PR TITLE
Version number added in item

### PR DESCRIPTION
**Pull request by Aca-jov**
SDNTB-115

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

File: scripts/superdesk-archive/views/media-box-list.html [hunk 1]
```diff
@@ -38,6 +38,7 @@
         </div>
         <div class="line">
             <div sd-item-state data-state=item.state data-embargo=item.embargo ng-hide="viewType !== undefined && viewType !== null && viewType !== 'deskOutput'"></div>
+            <div class="provider" ng-if="item.correction_sequence">Update {{ item.correction_sequence }}</div>
```
Comment (ahilles107) [inline scripts/superdesk-archive/views/media-box-list.html]: `Update` word is not translated?

Comment (Aca-jov) [inline scripts/superdesk-archive/views/media-box-list.html]: Its changed, translation added

**Comment by ahilles107** [comment]
:+1:

**Pull request merged**